federated-xgboost/README.md

```diff
@@ -3,10 +3,29 @@ Federated learning allows multiple parties to collaboratively learn a shared mod
 
 In the federated setting, a central party has a basic model that is initially broadcast to all parties. Each party locally trains the model with its own data, then sends a summary of the updates to the model back to the central party. In the decision tree case, parties would be sending the best local feature splits back to the central party. The central party then aggregates all updates, updates its own model with the aggregated update, and broadcasts the newly updated model to all parties. This process is then repeated over and over.
```

Comment:
Can you add a note here to look at the diagrams you've added for more information about splits and aggregation?